.github/workflows/auto-build.yml

```diff
@@ -0,0 +1,249 @@
+name: Auto build image
+on:
+  issue_comment:
+    types:
+      - created
+jobs:
+  issue_comment:
+    name: Auto build image
+    if: ${{ github.event.comment.body == '/buildv1.20.14' }}
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+
+      - name: Check outputs
+        run: |
+          BUILD=${{github.event.comment.body}}
+          echo ${BUILD#*d}
+          wget http://sealer.oss-cn-beijing.aliyuncs.com/auto-build/k8sv1.20.14.tar.gz
+          tar -xvf k8sv1.20.14.tar.gz
+          wget https://dl.k8s.io/${BUILD#*d}/kubernetes-server-linux-amd64.tar.gz
+          tar -xvf kubernetes-server-linux-amd64.tar.gz
+          sudo cp ./kubernetes/server/bin/kubectl ./rootfs/bin/
+          sudo cp ./kubernetes/server/bin/kubeadm ./rootfs/bin/
+          sudo cp ./kubernetes/server/bin/kubelet ./rootfs/bin/
+          wget http://sealer.oss-cn-beijing.aliyuncs.com/auto-build/sealer.tar.gz
+          sudo tar -xvf sealer.tar.gz -C /usr/bin
+          sudo sealer login ${{secrets.PRIVATEWAREHOUSE}}
+          cd ./rootfs
+          cat>imageList<<EOF
```

Comment:
![image](https://user-images.githubusercontent.com/69408038/150278671-27c9bc50-2e26-4b50-a56a-2fa6218b8cf1.png)

Reply:
Yes  you should use kubeadm to list all images.